README.txt

```diff
@@ -73,7 +73,7 @@ First step, you have to build the Docker image
 $ docker build -t james/project dockerfiles/compilation/java-8
 
 In order to run the build, you have to launch the following command:
-$ docker run -v $PWD/.m2:/root/.m2 -v $PWD:/origin -v $PWD/dockerfiles/run/spring/destination:/destination -t james/project -s SHA1
+$ docker run -v $PWD/.m2:/root/.m2 -v $PWD:/origin -v $PWD/dockerfiles/run/guice/destination:/destination -t james/project -s SHA1
```

Comment:
Then the line : 

> - $PWD/dockerfiles/run/spring/destination:/destination: is the third volume used to get the compiled elements, as it is needed by the container that will run James.

Should be more clear and state something like : 

```
 - $PWD/dockerfiles/run/spring/destination:/destination: is the third volume used to get the compiled elements, as it is needed by the container that will run James. Use $PWD/dockerfiles/run/spring/destination if you want to run James in a spring flavour, or $PWD/dockerfiles/run/guice/destination if you want to run it in a guice flavour.
```

Because like this, it is misleading. People following the instructions might export the build result to spring folder instead of guice.

What do you think ?